src/pkg/services/m365/api/team_test.go

```diff
@@ -0,0 +1,55 @@
+package api_test
+
+import (
+	"testing"
+
+	"github.com/alcionai/corso/src/internal/common/ptr"
+	"github.com/alcionai/corso/src/internal/tester"
+	"github.com/alcionai/corso/src/internal/tester/tconfig"
+	"github.com/alcionai/corso/src/pkg/fault"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type TeamsUnitSuite struct {
+	tester.Suite
+}
+
+func TestTeamsUnitSuite(t *testing.T) {
+	suite.Run(t, &TeamsUnitSuite{Suite: tester.NewUnitSuite(t)})
+}
+
+type TeamsIntgSuite struct {
+	tester.Suite
+	its intgTesterSetup
+}
+
+func TestTeamsIntgSuite(t *testing.T) {
+	suite.Run(t, &TeamsIntgSuite{
+		Suite: tester.NewIntegrationSuite(
+			t,
+			[][]string{tconfig.M365AcctCredEnvs}),
+	})
+}
+
+func (suite *TeamsIntgSuite) SetupSuite() {
+	suite.its = newIntegrationTesterSetup(suite.T())
```

Comment:
integrationTesterSetup should have teams (and groups?) details added to it, so that the data is available for all tests as needed.

Reply:
Added helper functions.